### PR TITLE
docs: Add section on how to force reconcile HRs

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-85660

## Description of changes being made
Add command on how to force HR reconciliation in case it is "stuck" in a failed state 

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.1/install/networked/index.md [hunk 1]
```diff
@@ -96,6 +96,12 @@ helmrelease.helm.toolkit.fluxcd.io/traefik-forward-auth-mgmt condition met
 helmrelease.helm.toolkit.fluxcd.io/velero condition met
 ```
 
+If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following command:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/install/networked/index.md:99]: Should we add something about how to find helmrelease and standup?

```suggestion
If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress" (to check the `HelmReleases` with statuses, use `kubectl get helmreleases -A`), you can trigger a reconciliation of the `HelmRelease` using the following command:
```

File: pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md [hunk 2]
```diff
@@ -138,6 +138,12 @@ Before running the following command, ensure that your `dkp` configuration **ref
     dkp upgrade kommander -v 4
     ```
 
+    If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following command:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md:141]:
```suggestion
    If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress" (to check the `HelmReleases` with statuses, use `kubectl get helmreleases -A`), you can trigger a reconciliation of the `HelmRelease` using the following command:
```

File: pages/dkp/kommander/2.2/install/networked/index.md [hunk 3]
```diff
@@ -94,6 +94,12 @@ helmrelease.helm.toolkit.fluxcd.io/traefik-forward-auth-mgmt condition met
 helmrelease.helm.toolkit.fluxcd.io/velero condition met
 ```
 
+If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following command:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/install/networked/index.md:97]:
```suggestion
If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress" (to check the `HelmReleases` with statuses, use `kubectl get helmreleases -A`), you can trigger a reconciliation of the `HelmRelease` using the following command:
```

**Review by msdolbey — APPROVED** [review]
Other than my nit (unnecessary probably) - looks good to me: 

reconciliation

File: pages/dkp/kommander/2.1/install/networked/index.md [hunk 4]
```diff
@@ -96,6 +96,12 @@ helmrelease.helm.toolkit.fluxcd.io/traefik-forward-auth-mgmt condition met
...
+
+```bash
+kubectl annotate --overwrite helmrelease/<HELMRELEASE_NAME> -n <WORKSPACE_NAMESPACE> reconcile.fluxcd.io/requestedAt="$(date +%s)"
```
Comment (makkes) [inline pages/dkp/kommander/2.1/install/networked/index.md:102]: This does not work if the number of retries is exhausted. See https://github.com/fluxcd/helm-controller/issues/366

I believe the suspend/resume trick works in that case but haven't tried.

Comment (gracedo) [reply pages/dkp/kommander/2.1/install/networked/index.md:102]: ah, thanks @makkes, will update

Comment (gracedo) [reply pages/dkp/kommander/2.1/install/networked/index.md]: Updated in https://github.com/mesosphere/dcos-docs-site/pull/4310/commits/edd9bc416d22f30b43cfc5fbda38493433cef88b

File: pages/dkp/kommander/2.2/install/networked/index.md (hunk 3 above)
Comment (gracedo) [reply pages/dkp/kommander/2.2/install/networked/index.md]: made a new section to check HR in https://github.com/mesosphere/dcos-docs-site/pull/4310/commits/edd9bc416d22f30b43cfc5fbda38493433cef88b

File: pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md [hunk 5]
```diff
@@ -138,6 +138,12 @@ Before running the following command, ensure that your `dkp` configuration **ref
...
+
+    ```bash
+    kubectl annotate --overwrite helmrelease/<HELMRELEASE_NAME> -n <WORKSPACE_NAMESPACE> reconcile.fluxcd.io/requestedAt="$(date +%s)"
```
Comment (takirala) [inline pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md:144]: i guess even this instruction needs to be switched to use `suspend` field?

Comment (gracedo) [reply pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md:144]: woops yeah missed this one thanks

**Pull request merged**